resources/assets/less/bem/stg-header.less

```diff
@@ -18,7 +18,7 @@
 
 .stg-header {
   .default-box-shadow();
-  background-image: url('/images/headers/community-support.png');
```

Comment:
looks like whoever wrote this initially forgot to use `.at2x-simple()` instead of this.